dotcom-rendering/package.json

```diff
@@ -50,6 +50,7 @@
 		"@guardian/libs": "19.2.1",
 		"@guardian/ophan-tracker-js": "2.2.5",
 		"@guardian/react-crossword": "2.0.2",
+		"@guardian/react-crossword-next": "npm:@guardian/react-crossword@0.0.0-canary-20250114144251",
```

Comment:
Would be preferable to use a release version rather than a canary, particularly before we move to a >0% size test.